### PR TITLE
added routing for Nav

### PR DESCRIPTION
**Pull request by Kolezhanchik**

### CODE REVIEW

File: components/nav/Nav.tsx [hunk 1]
```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
 import Logo from '@/components/assets/logo.svg';
 import ProfileDropdown from '@/components/nav/ProfileDropdown';
 import Link from 'next/link';
+import { headers } from 'next/headers';
+
+export default async function NavBar() {
+  const headersList = headers();
+  const referer = headersList.get('referer');
+  let url = '/private-cloud/products/all';
+  if (typeof referer === 'string') {
```
Comment (junminahn) [inline components/nav/Nav.tsx:11]: let's use `import _isString from 'lodash-es/isString` for better readability.

File: components/nav/Nav.tsx [hunk 2]
```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
...
+    if (referer.split('/')[3] === 'public-cloud') {
```
Comment (junminahn) [inline components/nav/Nav.tsx:12]: let's create a variable on top to reuse the variable for two conditions.

File: components/nav/Nav.tsx [hunk 3]
```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
...
+      url = '/public-cloud/products/all';
+    }
+    if (referer.split('/')[3] === 'private-cloud') {
```
Comment (junminahn) [inline components/nav/Nav.tsx:15]: let's use `else if`.

File: components/nav/Nav.tsx (hunk 2 above)
Comment (Kolezhanchik) [reply components/nav/Nav.tsx]: currentURL variable is created

File: components/nav/Nav.tsx (hunk 1 above)
Comment (Kolezhanchik) [reply components/nav/Nav.tsx]: changed to using _isString

File: components/nav/Nav.tsx (hunk 3 above)
Comment (Kolezhanchik) [reply components/nav/Nav.tsx]: doesn't work properly with `else if`

Comment (junminahn) [reply components/nav/Nav.tsx]: hm.. not sure why though. can you explain little more?

Comment (Kolezhanchik) [reply components/nav/Nav.tsx]: also changed, thank you

**Review by junminahn — APPROVED** [review]

**Pull request merged**